src/client/src/app/site/[site]/details/edit-details/edit-details-form.tsx

```diff
@@ -60,43 +61,70 @@ const EditDetailsForm = ({
 
   return (
     <form onSubmit={handleSubmit(submitForm)}>
-      <FormGroup error={errors.name?.message}>
+      <FormGroup
+        error={errors.name ? 'You have not entered a name' : undefined}
+      >
```

Comment:
What's the reason for this change? Why not just set the error message during the validation?